### PR TITLE
Create React app that says "Hello World!"

### PR DESCRIPTION
**Pull request by bonniezhou**
Used `create-react-app` to create basic Hello World! page. Use `npm run start` to run it locally.

This PR:
- includes `package.json` setup for GitHub pages and `.gitignore`
- still has a bunch of the React placeholder content (e.g. favicon) which is to be replaced in later PRs

Next PR will take care of deploying to GitHub pages as per [these instructions](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#step-2-install-gh-pages-and-add-deploy-to-scripts-in-packagejson).

Refs #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-scripts": "1.1.1"
+  },
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
```
Comment (lynnjepsen) [inline package.json]: was this all generated by create-react-app? I've never seen deploy and predeploy actually used.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "homepage": "https://material-components.github.io/material-components-web-catalog",
+  "devDependencies": {
+    "gh-pages": "^1.1.0"
```
Comment (lynnjepsen) [inline package.json]: You can wait to add this in the next PR

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (lynnjepsen) [inline public/index.html]: Change this to Material Component Web Catalog

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (lynnjepsen) [inline public/manifest.json]: Update the names here to MDC Web something

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.App {
```
Comment (lynnjepsen) [inline src/App.css]: make this sass file, then check that everything still loads through webpack. You might need sass-loader

File: src/index.css [hunk 6]
```diff
@@ -0,0 +1,5 @@
+body {
```
Comment (lynnjepsen) [inline src/index.css]: Could we put this in App.css?

File: yarn.lock [hunk 7]
```diff
@@ -0,0 +1,7241 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (lynnjepsen) [inline yarn.lock]: delete this, run `npm install` and make sure a package-lock.json is generated

File: package.json (hunk 1 above)
Comment (bonniezhou) [reply package.json]: This is all stuff for Github pages, I'll remove it and add it in the next PR

File: src/index.css (hunk 6 above)
Comment (bonniezhou) [reply src/index.css]: Yeah, I deleted index.css too

File: src/App.css (hunk 5 above)
Comment (bonniezhou) [reply src/App.css]: It still seems to work even without ejecting webpack... maybe webpack already accounts for sass-loader and we can actually avoid ejecting webpack altogether.

**Pull request merged**